CHANGELOG.md

```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.0]
+### Added
+- Add subpath exports ([#214](https://github.com/MetaMask/eth-sig-util/pull/214), [#211](https://github.com/MetaMask/eth-sig-util/pull/211))
+- Add `signTypedData` version validation ([#201](https://github.com/MetaMask/eth-sig-util/pull/201))
+- Add validation to check that parameters aren't nullish ([#205](https://github.com/MetaMask/eth-sig-util/pull/205))
```

Comment:
I'm not sure if these should be listed as `Added`, since they're changes we made to a function that's never been released before.

Reply:
Well, kinda :thinking: We had most of these functions already. Just that now the parameters they take are a different shape.

I didn't list this as breaking because I made sure that passing in a nullish value for the old versions of each parameter never worked anyway. But it might still be fair to list it as a change, in that we now have nicer error messages for these cases.

Reply:
This is probably better off under `Changed` than `Added` though. Same for the version validation. It's an addition to an already-existent thing.

Reply:
Works for me!

Reply:
Done in https://github.com/MetaMask/eth-sig-util/pull/217/commits/4a08cf7667a2731161e4665612fe2f9f561aea2a